OpenOversight/app/utils.py

```diff
@@ -361,6 +361,7 @@ def filter_by_form(form_data, officer_query, department_id=None):
         ]
 
         if "Not Sure" in form_data["unit"]:
+            # Convert "Not Sure" to None, so the resulting SQL query allows NULL values
```

Comment:
This is kind of confusing to me... should it _replace_ `"Not Sure"`? Cause at this point we're just adding `None` to allow null values... and I suppose `Not Sure` will never match anything, so the `in` query still works fine, right? So this is good 😂

Reply:
I think so? :sweat_smile: I'm no so familiar with this code...but yea I think it works as you describe!